monodbassignment/posts.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id":"String datatype", 
+  "author_id": "String datatype(id of author)",
+   "timestamp":"timestamp datatype",
+  "content":{
+    "head":"String datatype",
+    "body":"String datatype",
+    "tags":[],
+    "images":"url"
+    },
+    "comments":[{
+      "content":"String datatype",
+      "commenter_id":"String datatype",
+      "reply":{
```

Comment:
- key name should be replies, since you could have multiple replies
- should be an array of documents, not a single document.